C#/fgt_sdk_csharp/fgt_sdk.csproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>Fluigent</Authors>
+    <Product>Fluigent Software Development Kit</Product>
+    <Description>Software Development Kit for Fluigent pressure controllers and sensors.</Description>
+    <Version>19.0.1</Version>
+    <PackageTags>Microfluidics, Control</PackageTags>
+    <Platforms>AnyCPU;x64;x86</Platforms>
```

Comment:
If you plan to build for x86, make sure you have the .net core x86 installed and override the default "RunCommand": https://github.com/dotnet/core/issues/901#issuecomment-328307413